.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: node_js
+dist: precise
+cache: yarn
+sudo: required
+addons:
+  firefox: "31.0"
+  phantomjs: "1.9.8"
```

Comment:
Do these changes fix something, or is travis still broken with them? I wonder if they'll just end up muddying the waters for getting travis fixed if they're not doing something....

Reply:
It is still broken, but less broken (it runs a few tests before timing out, as opposed to just instantly timing out)

Reply:
could just remove